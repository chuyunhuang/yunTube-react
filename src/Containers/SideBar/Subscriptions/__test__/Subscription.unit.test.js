import React from 'react';
import { shallow } from 'enzyme';
import { Subscription } from '../Subscription';

describe('Subscribtion', () => {
  test('renders', () => {
    const wrapper = shallow(
      <Subscription />
    );
    expect(wrapper).toMatchSnapshot();
  });
});