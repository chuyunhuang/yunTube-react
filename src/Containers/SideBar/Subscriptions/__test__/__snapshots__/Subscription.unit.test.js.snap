// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubsciptionGroup render broadcasting Subscription 1`] = `
<Fragment>
  <SideBarHeader
    title="Subscriptions"
  />
  <Subscription
    broadcasting={true}
    label="Music Channel"
  />
  <Subscription
    amountNewVideos={10}
    label="Travel Channel"
  />
  <Subscription
    amountNewVideos={10}
    label="Ted x Talks"
  />
  <Divider />
</Fragment>
`;

exports[`SubsciptionGroup renders an empty SubscriptionGroup 1`] = `
<Fragment>
  <SideBarHeader
    title="Subscriptions"
  />
  <Subscription
    broadcasting={true}
    label="Music Channel"
  />
  <Subscription
    amountNewVideos={10}
    label="Travel Channel"
  />
  <Subscription
    amountNewVideos={10}
    label="Ted x Talks"
  />
  <Divider />
</Fragment>
`;

exports[`SubsciptionGroup renders non-broadcasting Subscription with new videos 1`] = `
<Fragment>
  <SideBarHeader
    title="Subscriptions"
  />
  <Subscription
    broadcasting={true}
    label="Music Channel"
  />
  <Subscription
    amountNewVideos={10}
    label="Travel Channel"
  />
  <Subscription
    amountNewVideos={10}
    label="Ted x Talks"
  />
  <Divider />
</Fragment>
`;

exports[`Subscribtion renders 1`] = `
<MenuItem>
  <div
    className="subscription"
  >
    <div>
      <Image
        as="img"
        avatar={true}
        src="http://via.placeholder.com/28x28"
        ui={true}
      />
      <span />
    </div>
  </div>
</MenuItem>
`;
